ckanext/bcgov/util/helpers.py

```diff
@@ -366,12 +367,16 @@ def get_edc_org(org_id):
 
 def get_orgs_form(field = None):
     """Designed to get available orgs for scheming fields as parameters cannot be defined in choices_helper functions"""
-    orgs = []
-    for org in ckan.lib.helpers.organizations_available('create_dataset'):
-        orgs.append({
-            'value': org['id'],
-            'label': org['display_name']
-        })
+    context = {'model': model, 'session': model.Session,
+               'user': c.user, 'auth_user_obj': c.userobj}           
+    data_dict = {
+        'is_organization': True,
+        'all_fields': True
+    }
+    all_orgs_data = (organization_or_group_list_related(context, data_dict))
+
+    orgs = [{'id': org['id'], 'display_name': org['display_name']} for org in all_orgs_data]
```

Comment:
```suggestion
    orgs = [{'value': org['id'], 'label': org['display_name']} for org in all_orgs_data]
```
Should it not be like this to return same object values as the original?

Reply:
Yes, it should be. I will fix that